packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/integrationsteps.md

```diff
@@ -1,121 +1,123 @@
+After the user verifies their identity using the email authenticator, `OktaAuth.idx.proceed()` returns an `IdxTransaction` indicating:
 
-<div class="common-image-format">
+* The sign-in `status` is in the `PENDING` state.
+* The next step is to reset the user's password, that is, `nextStep.name` = `reset-authenticator`.
 
-![Displays the Challenge email authenticator page that has a field to input your email verification code.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-challenge-nodejs.png)
-
-</div>
-
-After the user checks their email for the code and enters the value into the field, they click a button to initiate the verification. Call `idx.recoverPassword` with the verification code passed in, as shown in the `authenticator.js` page of the SDK sample application.
-
-```JavaScript
-router.post('/challenge-authenticator/email', async (req, res, next) => {
-  const { idxMethod } = req.getFlowStates();
-  const { verificationCode } = req.body;
-  const authClient = getAuthClient(req);
-  const transaction = await authClient.idx[idxMethod]({ verificationCode });
-  handleTransaction({ req, res, next, authClient, transaction });
-});
+```javascript
+{
+  status: "PENDING",
+  nextStep: {
+    name: "reset-authenticator",
+    inputs: [
+      {
+        name: "password",
+        label: "New password",
+        secret: true,
+        type: "string",
+        required: true,
+      },
+    ],
+  },
+}
 ```
 
-If the `idx.recoverPassword` call is successful, it returns a status of `Idx.Status:PENDING` and the `nextStep` field indicates an input of a password value parameter. This status indicates that the user can now change their password. At this point, the user is redirected to a change password page.
-
-Create a reset password page that allows the user to enter the new password, confirm it, and initiate the update.
+Create a password reset page that allows the user to enter their new password.
 
-<div class="common-image-format">
+<div class="common-image-format bordered-image">
 
-![Displays the Reset Password page that has a field to input your password and a field to confirm the password value.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-set-password-nodejs.png)
+![Screenshot of a page allowing the user to enter their new password](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-set-password-nodejs.png)
 
 </div>
 
-### 5: Add the new password
+### 7. Submit the new password
 
-The final step is to make a call to `idx.recoverPassword`, passing in the value of the user input password, as shown in the `recover-password.js` file of the SDK sample application. This call resets the user's password.
+When the user submits their new password, pass it as a parameter to `OktaAuth.idx.recoverPassword()`.
```

Comment:
I think you need to include this code you have deleted and note that the two password fields should contain the same value before passing the password to Okta. If you don't, you're effectively ignoring the "Reneter password" box and why its there.